src/components/DocumentationTopic/Summary/Availability.vue

```diff
@@ -119,72 +124,43 @@ export default {
 .tech-icon {
   height: 12px;
   padding-right: 5px;
-  fill: var(--badge-color);
-
-  @include prefers-dark {
-    fill: var(--badge-dark-color);
-  }
-}
-
-.beta {
-  color: var(--color-badge-beta);
-
-  @include prefers-dark {
-    color: var(--color-badge-dark-beta);
-  }
-}
-
-.deprecated {
-  color: var(--color-badge-deprecated);
-
-  @include prefers-dark {
-    color: var(--color-badge-dark-deprecated);
-  }
+  fill: var(--color-figure-gray) !important;
```

Comment:
Can we do this in a way that doesn't need `!important`?

Haven't tried these, but these are the workarounds I might try first:
* make sure your override has the right precedence
* possibly override `--color-svg-icon` here instead